content/en/blog/2026-06-10-etcd-operator-joins-cozystack.md

```
@@ -0,0 +1,73 @@
+date: 2026-06-10
+author: "Andrey Kolkov (Ænix), Andrei Kvapil (Ænix)"
+description: "The etcd-operator project has been donated to Cozystack, together with a from-scratch implementation under the new etcd-operator.cozystack.io/v1alpha2 API that drives etcd's native Membership API instead of a StatefulSet."
+article_types:
+  - news
+topics:
+  - platform
+  - kubernetes
+
+---
+
+### etcd-operator Joins Cozystack with a New v1alpha2 API
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)